### PR TITLE
Add new ChangeLog Updates

### PR DESCRIPTION
**Pull request by confused-Techie**
Like the title says, this PR adds the newest changelog updates to both the `CHANGELOG.md` and `welcome` package ChangeLog view

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,53 @@
...
+- Added: Decaf `wrap-guide` [@confused-Techie](https://github.com/pulsar-edit/pulsar/pull/443)
+- Added: Additional Bundling of Core Packages [@confused-Techie](https://github.com/pulsar-edit/pulsar/pull/424)
+- Added: add allow-jit entitlement (fixes Apple Silicon builds) [@Meadowsys](https://github.com/pulsar-edit/pulsar/pull/454)
+- Removed: Revert "Create i18n API" [@mauricioszabo](https://github.com/pulsar-edit/pulsar/pull/471)
+- Added: Build first, and test later [@mauricioszabo](https://github.com/pulsar-edit/pulsar/pull/463)
+- Update: [settings-view] Update package snippets view to reflect new features [@savetheclocktower](https://github.com/pulsar-edit/pulsar/pull/406)
+- Added: Create i18n API [@Meadowsys](https://github.com/pulsar-edit/pulsar/pull/446)
+- Added: Add Automated updating of `autocomplete-html` `completions.json` [@confused-Techie](https://github.com/pulsar-edit/pulsar/pull/405)
+- Fixed: docs: fix markdown links in packages README [@oakmac](https://github.com/pulsar-edit/pulsar/pull/450)
+- Fixed: Path document register element [@mauricioszabo](https://github.com/pulsar-edit/pulsar/pull/438)
```
Comment (DeeDeeG) [inline CHANGELOG.md:31]:
```suggestion
- Fixed: Patch document register element [@mauricioszabo](https://github.com/pulsar-edit/pulsar/pull/438)
```

Typo "Path" --> "Pat**c**h"

**Review by DeeDeeG — APPROVED** [review]
LGTM.

 I only really verified that the "\### Pulsar" section of CHANGELOG.md matches the apparent merged PR's since 1.103.0. But that seems to match, very nice!

(Based on this comparison view, visually scanning for the hyperlinks in the commit messages that show which PR was merged... https://github.com/pulsar-edit/pulsar/compare/v1.103.0...master)

And I expect we will be including https://github.com/pulsar-edit/pulsar/pull/451 soon, let's not forget to add that in to the changelogs if it gets merged before release.

File: CHANGELOG.md (hunk 1 above)
Comment (confused-Techie) [reply CHANGELOG.md:31]: Great catch here thanks!

**Review by DeeDeeG — APPROVED** [review]
Re-approved, with the PR 451 line now added since that PR just merged. 👍

**Comment by confused-Techie** [comment]
As PR is out to bump the tagged version, I'll go ahead and merge this one, thanks for the reviews!

**Pull request merged**